src/preflight/handler.js

```diff
@@ -242,25 +310,83 @@ export const preflightAudit = async (context) => {
         }));
 
       if (insecureLinks.length > 0) {
-        auditsByName[AUDIT_LINKS].opportunities.push({ check: 'bad-links', issue: insecureLinks });
+        const audit = pageResult.audits.find((a) => a.name === AUDIT_LINKS);
+        audit.opportunities.push({ check: 'bad-links', issue: insecureLinks });
       }
     });
+    const domEndTime = Date.now();
+    const domEndTimestamp = new Date().toISOString();
+    const domElapsed = ((domEndTime - domStartTime) / 1000).toFixed(2);
+    log.info(`[preflight-audit] site: ${site.getId()}, job: ${jobId}, step: ${normalizedStep}. DOM-based audit completed in ${domElapsed} seconds`);
+
+    await saveIntermediateResults(result, 'DOM-based audit');
+
+    const endTime = Date.now();
+    const endTimestamp = new Date().toISOString();
+    const totalElapsed = ((endTime - startTime) / 1000).toFixed(2);
+    log.info(`[preflight-audit] site: ${site.getId()}, job: ${jobId}, step: ${normalizedStep}. Total audit time: ${totalElapsed} seconds`);
+    log.info(`[preflight-audit] site: ${site.getId()}, job: ${jobId}, step: ${normalizedStep}. Audit started at: ${startTimestamp}`);
```

Comment:
nit:
@slitviachenko instead of pasting the timestamp in the logs, should we add one log print in the start and one in the end ?
My concern is does it look sophisticated in the log prints.

Reply:
Yeah, not sure we benefit much from that summary in the logs, so I removed it.